### ISSUE TITLE
panic: interface conversion: interface {} is float64, not []uint8

### ISSUE THREAD

**Issue by matteraf** [martinarrieta/go-dump#12]
Hi, 

I got this error when I run the command:

```code
goroutine 2778 [running]:
github.com/martinarrieta/go-dump/go/utils.(*DataChunk).Parse(0xc420a53e10, 0xc42072a900, 0xc4201384e0, 0x7, 0xc420a53da0)
	/home/travis/gopath/src/github.com/martinarrieta/go-dump/go/utils/datachunk.go:134 +0xc99
github.com/martinarrieta/go-dump/go/utils.(*TaskManager).StartWorker(0xc420142000, 0x5)
	/home/travis/gopath/src/github.com/martinarrieta/go-dump/go/utils/taskmanager.go:433 +0xe7
created by github.com/martinarrieta/go-dump/go/utils.(*TaskManager).StartWorkers
	/home/travis/gopath/src/github.com/martinarrieta/go-dump/go/utils/taskmanager.go:350 +0xf3
```

this is the command line

```code
./go-dump-v0.0.5-linux-amd64 --mysql-socket=/var/lib/mysql/mysql.sock --mysql-user=root --mysql-password=root --destination=/tmp/exports --all-databases  --execute=true --tables-without-uniquekey=single-chunk --isolation-level="READ COMMITTED" --consistent=false --lock-tables=false --master-data=false --threads=8 --compress=true --compress-level=5
```

**Comment by martinarrieta** [comment]
Thank you for the report! 

As we discussed, the issue was a double field that was converted to float64. I have fixed that issue in the release 0.0.6.

Regards, 

Martin

**Comment by martinarrieta** [comment]
Hi, I'm going to close this issue.

Thanks again for submit the error.

Regards, 

Martin.
